jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -111,6 +123,12 @@ public String getJarPluginName() {
     return JAR_PLUGIN_NAME;
   }
 
+  @Override
+  public boolean isWarProject() {
+    // TODO: replace with "getWar(project) != null" once ready
```

Comment:
Is this ready?

Reply:
I'll use this as a feature flag during development, and make it work only when all the parts are working.